port/unix_include/omrsock.h

```diff
@@ -41,6 +41,7 @@
 #include <sys/types.h> /* Some historical implementations need this file, POSIX.1-2001 does not. */
 #include <sys/socket.h>
 #include <fcntl.h>
+#include <poll.h>
```

Comment:
does this need to be sys/poll.h on aix?

Reply:
I will run on omr jenkins.

Reply:
It seems that just having poll.h is fine.